Better error messages when kubeconfig file not present.

Ex: running `ks init foo` without the kubeconfig file present.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1b2b126]

goroutine 1 [running]:
github.com/ksonnet/ksonnet/cmd.resolveContext(0x0, 0xc4202eded0, 0x7, 0x0, 0x0, 0x0, 0x0)
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/cmd/root.go:151 +0x296
github.com/ksonnet/ksonnet/cmd.glob..func9(0x251d0c0, 0xc4202ede80, 0x1, 0x1, 0x0, 0x0)
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/cmd/init.go:68 +0x237
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).execute(0x251d0c0, 0xc4202ede40, 0x1, 0x1, 0x251d0c0, 0xc4202ede40)
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:656 +0x3f1
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x251f920, 0xc420533f10, 0x1, 0x1)
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:735 +0x2fe
github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra.(*Command).Execute(0x251f920, 0x0, 0x5b)
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/vendor/github.com/spf13/cobra/command.go:694 +0x2b
main.main()
    /Users/jyuen/workspace/go/src/github.com/ksonnet/ksonnet/main.go:33 +0x5b

--- a/cmd/root.go
+++ b/cmd/root.go
@@ -148,7 +148,12 @@ func resolveContext(context *string) (server, namespace string, err error) {
 	}
 	ctx := rawConfig.Contexts[ctxName]
 	if ctx == nil {
-		return "", "", fmt.Errorf("context '%s' does not exist in the kubeconfig file", *context)
+		if len(ctxName) == 0 && ctxName == rawConfig.CurrentContext {
+			// User likely does not have a kubeconfig file.
+			return "", "", fmt.Errorf("No current context found. Make sure a kubeconfig file is present")
+		}
+
+		return "", "", fmt.Errorf("context '%s' does not exist in the kubeconfig file", ctxName)
 	}
 
 	log.Infof("Using context '%s'", ctxName)

